jsx-runtime/src/index.js

```diff
@@ -35,11 +35,10 @@ function createVNode(type, props, key, __source, __self) {
 		_component: null,
 		_hydrating: null,
 		constructor: undefined,
-		_original: undefined,
+		_original: Math.random(),
```

Comment:
This will transition the field from SMI

Reply:
What do you mean?

Reply:
SMI = small integer, it's a data type inside v8 https://v8.dev/blog/elements-kinds